lib/i18n/hygiene/checks/html_entity.rb

```diff
@@ -0,0 +1,32 @@
+require 'i18n/hygiene/checks/base'
+require 'i18n/hygiene/keys_with_entities'
+require 'i18n/hygiene/result'
+require 'i18n/hygiene/wrapper'
+
+module I18n
+  module Hygiene
+    module Checks
+      class HtmlEntity < Base
+        def run
+          puts "Checking for phrases that contain entities but probably shouldn't..."
+
+          wrapper = I18n::Hygiene::Wrapper.new(locales: config.locales)
```

Comment:
if `config.locales` contains `[:fr,:es]`, does that mean this task won't check the main locale (`:en` for us) as well?